packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -89,6 +89,12 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
     assert(!attached);
   }
 
+  /// Find the last [AnnotatedRegionLayer] which contains [regionOffset] with a
+  /// value of Type [type].
+  ///
+  /// Returns null if no matching region is found.
+  Object findRegion(Offset regionOffset, Type type);
```

Comment:
The first sentence of these docs shouldn't mention AnnotatedRegionLayer, because technically any Layer could implement this to return data. I suggest we say something like "Returns the value of type `type` that corresponds to the point described by `regionOffset`.", or some such. Then you can mention in a second paragraph that AnnotatedRegionLayer is the main way for a value to be assigned here or some such.